contracts/LockedFund/LockedFund.sol

```diff
@@ -386,6 +477,39 @@ contract LockedFund is ILockedFund {
 		emit WaitedTSUpdated(msg.sender, _waitedTS);
 	}
 
+	/**
+	 * @notice The internal function used to update the Token.
+	 * @param _token The address of the ERC20 Token.
+	 */
+	function _changeToken(address _token) internal {
+		require(_token != address(0), "LockedFund: Invalid Token Address.");
```

Comment:
consider checking if the address is contract e.g. using openzeppelin Address.sol

Reply:
Solved in c47466e